### PR TITLE
Feature/edit message ph 4484

### PR DESCRIPTION
**Pull request by itdsntwork**
This pr adds the functionality to be able to edit or delete your own messages.

### CODE REVIEW

File: lib/src/message-list/message-editor.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  border-radius: 10%;
+  padding-right: 1rem;
+`;
+
+const MessageEditor = ({ onSubmit = () => {}, envelope }: Props) => {
+  const [value, setValue] = useState("");
+
+  console.log("asdfasdf", envelope);
+
+  React.useEffect(() => {
```
Comment (manoshatzi) [inline lib/src/message-list/message-editor.tsx]: please keep one way if you want. I mean you add the useState in the main declaration but the useEffect you call it through React.

**Review by manoshatzi — APPROVED** [review]
LGTM !

File: lib/src/message-list/message-editor.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  padding-left: 1rem;
+  background: "#5babfc" !important;
+  border-radius: 10%;
+  padding-right: 1rem;
+`;
+
+const MessageEditor = ({ onSubmit = () => {}, envelope }: Props) => {
+  const [value, setValue] = useState("");
+
+  console.log("asdfasdf", envelope);
```
Comment (stuckj) [inline lib/src/message-list/message-editor.tsx]: Can remove this console log.

**Review by stuckj — APPROVED** [review]
Tiny comment, but looks good.

**Pull request merged**